src/DataSource/DoctrineDataSource.php

```diff
@@ -72,7 +72,7 @@ public function getQuery(): Query
 
 
 	/********************************************************************************
-	 *                          IDataSource implementation                          *
+	 *                          IDataSource implementation *
```

Comment:
We can change that to some simple doc block....

Reply:
Solved